packages/auth/plugin/__tests__/iosPlugin_openUrlFix.test.ts

```diff
@@ -219,7 +244,9 @@ describe('Config Plugin iOS Tests - openUrlFix', () => {
         modRawConfig: { name: 'TestName', slug: 'TestSlug' },
       };
       const props = undefined;
-      const spy = jest.spyOn(WarningAggregator, 'addWarningIOS');
+      const spy = jest
+        .spyOn(WarningAggregator, 'addWarningIOS')
+        .mockImplementation(() => undefined);
```

Comment:
@mikehardy related to [Line 220](https://github.com/invertase/react-native-firebase/pull/8203/commits/7f0412e4a450696603bd1a2eac3957c21188c3c6#diff-0f4c13658fa7495aea4ea28afee8291a8e3c4ee58922219759045179709e67d1R220), when commenting this line `mockImplementation()` , why doesn't jest print a warning message with something like:
```
» ios: @react-native-firebase/auth: Skipping iOS openURL fix because no 'openURL' method was found
```